packages/runtime/container-runtime/src/id-compressor/persistanceUtilities.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable no-bitwise */
+import { NumericUuid } from "./identifiers";
+
+const biguint64Buffer = new BigUint64Array(2);
+const biguint64FloatView = new Float64Array(biguint64Buffer.buffer);
```

Comment:
NaN makes this lossy becaus ethere are many representations of NaN in ieee fp

Reply:
fixed by using BigUint64Array instead